auth/src/main/java/com/homihq/db2rest/auth/data/ApiAuthDataProvider.java

```diff
@@ -48,11 +48,4 @@ public List<User> getUsers() {
     public List<ApiExcludedResource> getExcludedResources() {
         return List.of();
     }
-
-    @Override
-    public Optional<User> getUserByUsername(String username) {
```

Comment:
Let's imagine our business logic changed we need to define **getUserByUsername** for the **ApiAuthDataProvider**. I suspect we are going to rollback the MR's changes.